yvalenta/parallel_search.rb

```diff
@@ -0,0 +1,46 @@
+class SearchVowels
+
+  def initialize
+    @time ||= {start: 0, finish: 0}
+  end
+
+  def open_file
+    f = File.open("../text.txt", "r")
+    run(f)
+    f.close
+  end
+
+  private
+
+  def vowels
+    @vowels ||= { 'a' => 0, 'e' => 0, 'i' => 0, 'o' => 0, 'u' => 0 }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.